### ISSUE TITLE
Not an issue. Just a comment

### ISSUE THREAD

**Issue by MJonesDev** [jparkie/Aizoban#49]
Hey, I am a fan of your work and I am saddened to see that my favorite Manga reader has been removed :'c
But I am glad to see that it is Open Source. So I have decided to make my own version of this app (Yes, I am crediting you, don't worry :) ) (unless you don't want me to. Then I will stop development on my version) but I will also contribute to this project and try to help out as much as I can. I am still learning code (main reason Why I am making my own based off of this code) but I am willing to help!

**Comment by jparkie** [comment]
I am totally fine if you wish to make your own version of the application. I ceased all the development since the DMCA request. Furthermore, because I am currently working full-time while I will start university this September, I don't think I will return to this project. Good luck! :)

Note: I recommend not following my schema for the database; it was not adaptable to changes in the sources. Finally, not all parts of the code is nice; this was my first Android application which I refactored through various versions from Alpha, Beta, V1, and an incomplete V2.
